components/plugins/PluginsLists.vue

```diff
@@ -90,19 +86,41 @@
     };
 
     const capitalize = (name: string) => {
-        return name[0].toUpperCase() + name.slice(1).toLowerCase();
+        return name[0].toUpperCase() + name.slice(1);
     };
 
     const totalGroups = computed(() => filteredPluginsData.value.length);
 
+    function tooltipContent(plugin: Plugin, filteredPluginElementsEntries: [string, PluginElement[]][]) {
+        return filteredPluginElementsEntries.map(([elementType, elements]) =>
+            `<p>${capitalize(elementType).replaceAll(/[A-Z]/g, match => ` ${match}`)}</p>
+<ul>
+${elements.map(({cls}) => `<li>
+                <a href="plugins/${plugin.title}/${cls}">${cls}</a>
+              </li>`).join("")}
+</ul>`).join("");
+    }
+
     const filteredPluginsData = computed(() => {
-        let pluginsList = [...props.plugins];
-        let searchResults = [];
-        if (searchQuery.value) {
-            searchResults = setSearchPlugins(searchQuery.value, pluginsList);
-        } else {
-            searchResults = pluginsList;
-        }
+        const filteredPlugins = props.plugins.flatMap(plugin => {
+            let filteredPluginElementsEntries = Object.entries(plugin)
```

Comment:
Why the sudden `let` here ?
```suggestion
            const filteredPluginElementsEntries = Object.entries(plugin)
```

Reply:
or from my research probably a copy paste so I repeated errors from the past :P